src/awkward/_v2/contents/numpyarray.py

```diff
@@ -405,7 +392,7 @@ def mergeable(self, other, mergebool):
         ):
             return self.mergeable(other._content, mergebool)
 
-        if isinstance(other, ak._v2.contents.numpyarray.NumpyArray):
+        elif isinstance(other, ak._v2.contents.NumpyArray):
```

Comment:
I agree that this should be chained with the if-elif above, since both of those return values. The code flow is the same either way, but we should adopt a default, and I prefer the if-elif-elif default.

Reply:
Yes, I'm following the Awkward code style of minimal early returns :)